### ISSUE TITLE
yolov8

### ISSUE THREAD

**Issue by sharonjunjun** [johnning2333/M2Doc#3]
when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.

**Comment by johnning2333** [comment]
> when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.

We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline.  While we encourage you to extend the m2doc to the yolo detector.

Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.

**Comment by carlos-vinicios** [comment]
> > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> 
> We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> 
> Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.

And the pretrained weights for the best model. You will provide too?

**Comment by johnning2333** [comment]
> > > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> > 
> > 
> > We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> > Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.
> 
> And the pretrained weights for the best model. You will provide too?

yes, we will also provide the pretrained weight of DINO and Cascade Mask R-CNN with m2doc on DocLayNet.

**Comment by sharonjunjun** [comment]
> > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> 
> We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> 
> Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.

ok. if i want to use DocLayNet dataset training, i should download DocLayNet_core and DocLayNet_extra get the object boundingbox and text boundingbox?thanks.

**Comment by johnning2333** [comment]
> > > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> > 
> > 
> > We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> > Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.
> 
> ok. if i want to use DocLayNet dataset training, i should download DocLayNet_core and DocLayNet_extra get the object boundingbox and text boundingbox?thanks.

yes

**Comment by sharonjunjun** [comment]
> > > > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> > > 
> > > 
> > > We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> > > Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.
> > 
> > 
> > ok. if i want to use DocLayNet dataset training, i should download DocLayNet_core and DocLayNet_extra get the object boundingbox and text boundingbox?thanks.
> 
> yes

ok,the inference script by onnx model will release and how long is the inference time? 
Can the model be deployed on triton?

**Comment by johnning2333** [comment]
> > > > > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> > > > 
> > > > 
> > > > We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> > > > Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.
> > > 
> > > 
> > > ok. if i want to use DocLayNet dataset training, i should download DocLayNet_core and DocLayNet_extra get the object boundingbox and text boundingbox?thanks.
> > 
> > 
> > yes
> 
> ok,the inference script by onnx model will release and how long is the inference time? Can the model be deployed on triton?

Thank you for your interest in our work. We currently do not have plans to release versions for ONNX or Triton

**Comment by johnning2333** [comment]
> > when can you add the yolo detector in the project and add the add the data format samples for M2Doc and add the dataset converting scripts? thanks.
> 
> We don't actually use yolov8 as our baseline for comparison. As we mentioned in the paper, we use DINO, Cascade Mask R-CNN for our main baseline. While we encourage you to extend the m2doc to the yolo detector.
> 
> Since we use mmdetection to train the models, we will provide the coco dataformat converting script and data samples in this month.

Sorry about the delay of the dataset format converting script.
We are now provide the ocr_anno_convert.py to format doclaynet ocr annotations, and we upload 3 test samples for illustration.
